enterprise/backend/test/metabase_enterprise/serialization/v2/load_test.clj

```diff
@@ -254,3 +255,91 @@
             (is (= 3 (db/count PulseChannelRecipient)))
             (is (= #{(:id @u1d) (:id @u2d) (:id @u3d)}
                    (db/select-field :user_id PulseChannelRecipient)))))))))
+
+(deftest card-dataset-query-test
+  ;; Card.dataset_query is a JSON-encoded MBQL query, which contain database, table, and field IDs - these need to be
+  ;; converted to a portable form and read back in.
+  ;; This test has a database, table and fields, that exist on both sides with different IDs, and expects a card that
+  ;; references those fields to be correctly loaded with the dest IDs.
+  (testing "embedded MBQL in Card :dataset-query is portable"
+    (let [serialized (atom nil)
+          coll1s     (atom nil)
+          db1s       (atom nil)
+          table1s    (atom nil)
+          field1s    (atom nil)
+          card1s     (atom nil)
+          user1s     (atom nil)
+          db1d       (atom nil)
+          table1d    (atom nil)
+          field1d    (atom nil)
+          user1d     (atom nil)
+          card1d     (atom nil)
+          db2d       (atom nil)
+          table2d    (atom nil)
+          field2d    (atom nil)]
+
```

Comment:
nit: extra line

Reply:
Submitting without this; the follow-up PR removes it.